app/pages/dashboard/contextCreator.test.js

```diff
@@ -16,20 +16,31 @@ describe('getOrgDashboardContext', () => {
   });
 
   it('should transform data into correct format if all data provided', () => {
-    const mockData = [{ name: 'Greater Manchester CCG', odsCode: 'X01' }, { name: 'Hampshire CCG', odsCode: 'X02' }];
+    const mockData = [{ name: 'Greater Manchester CCG', odsCode: 'X01', orgId: 'org1' }, { name: 'Hampshire CCG', odsCode: 'X02', orgId: 'org2' }];
     const { data } = getOrgDashboardContext(mockData);
-    expect(data[0][0]).toEqual(mockData[0].name);
-    expect(data[0][1]).toEqual(mockData[0].odsCode);
-    expect(data[1][0]).toEqual(mockData[1].name);
-    expect(data[1][1]).toEqual(mockData[1].odsCode);
+    expect(data[0][0].data).toEqual(mockData[0].name);
+    expect(data[0][0].href).toEqual(`organisations/${mockData[0].orgId}`);
+    expect(data[0][1].data).toEqual(mockData[0].odsCode);
+    expect(data[1][0].data).toEqual(mockData[1].name);
+    expect(data[1][0].href).toEqual(`organisations/${mockData[1].orgId}`);
+    expect(data[1][1].data).toEqual(mockData[1].odsCode);
   });
 
   it('should transform data into correct format if data provided has missing fields', () => {
-    const mockData = [{ odsCode: 'X01' }, { name: 'Hampshire CCG' }];
+    const mockData = [
+      { odsCode: 'X01', orgId: 'org2' },
+      { name: 'Hampshire CCG', orgId: 'org2' },
+      { name: 'Somerset CCG', odsCode: 'X01' },
+    ];
     const { data } = getOrgDashboardContext(mockData);
-    expect(data[0][0]).toEqual('');
-    expect(data[0][1]).toEqual(mockData[0].odsCode);
-    expect(data[1][0]).toEqual(mockData[1].name);
-    expect(data[1][1]).toEqual('');
+    expect(data[0][0].data).toEqual('');
+    expect(data[0][0].href).toEqual(`organisations/${mockData[0].orgId}`);
```

Comment:
Should we show the row at all if there is no `name` in the row object?

Reply:
yes to preserve spacing